frontend/src/views/Unauthorized.vue

```diff
@@ -1,16 +1,23 @@
 <template>
-  <div>
-    <h1>Access Denied</h1>
-    <p>We're sorry, but you do not have access to the requested resource</p>
-    <p>
-      Please contact the Help Desk if you require further assistance<br />
-      Phone number: xxx-xxx-xxxx<br />
-      Email: --@gov.bc.ca
-    </p>
-  </div>
+  <h1>Service Permissions not assigned</h1>
+  <p>Your log-in credentials are valid but you don't have Permissions to view or run transactions in MSP Direct. Please follow the instructions below to get permissions.</p>
```

Comment:
Permissions shouldn't be capitalized

Reply:
This is what's in the mockup.
![image](https://user-images.githubusercontent.com/80127167/162290309-b7c1214b-5754-4004-a9af-8781fc11887b.png)